### ISSUE TITLE
MAE with extra GAN loss

### ISSUE THREAD

**Issue by mehdidc** [facebookresearch/mae#9]
Hello,

thank you for releasing the code! I have seen that the Colab notebook  includes the visualization of an MAE trained with an extra GAN loss, which was not mentioned in the paper.
I was wondering, does it improve the  results (linear probe/fine-tuning) in any way ? if so, are you planning to release its code?

Thanks

**Comment by KaimingHe** [comment]
The GAN loss is only for generating more realistic visualization at the time being. Its influence on representation quality is to be studied. It is beyond the scope of this paper. We have not planned to release its code.

**Comment by mehdidc** [comment]
Thanks!

**Comment by LehaoLin** [comment]
Maybe this paper in CVPR2023 can meet your needs. But the code is not released by now. https://openaccess.thecvf.com/content/CVPR2023/papers/Fei_Masked_Auto-Encoders_Meet_Generative_Adversarial_Networks_and_Beyond_CVPR_2023_paper.pdf

**Comment by hellojialee** [comment]
> Maybe this paper in CVPR2023 can meet your needs. But the code is not released by now. https://openaccess.thecvf.com/content/CVPR2023/papers/Fei_Masked_Auto-Encoders_Meet_Generative_Adversarial_Networks_and_Beyond_CVPR_2023_paper.pdf

Has anymore reproduced this paper?
